hazelcast/src/main/java/com/hazelcast/jet/impl/execution/init/ExecutionPlanBuilder.java

```diff
@@ -279,10 +288,47 @@ public static Map<MemberInfo, int[]> getPartitionAssignment(
                     .add(partitionId);
         }
 
+        // Interactive prunable queries may require coordinator to be present
+        if (pruningLevels.contains(COORDINATOR_REQUIRED)) {
+            // If coordinator still not captured to participate in the job -- do it.
+            partitionsForMember.computeIfAbsent(localMemberInfo, (i) -> {
+                nodeEngine.getLogger(ExecutionPlanBuilder.class).fine("Adding coordinator to partition-pruned job members");
+                var partitionContainer = new FixedCapacityIntArrayList(partitionCount);
+                // If DAG has any other distributed-partitioned edge -> we will assign all partitions afterwards.
+                if (!pruningLevels.contains(ALL_PARTITIONS_REQUIRED)) {
+                    // TODO: temporarily we're using "" key.
```

Comment:
should this TODO be fixed in this PR? I assume that this handles `allToOne("")` but this is not the only allowed value for `allToOne` parameter

Reply:
outdated